compete.html

```diff
@@ -130,12 +130,12 @@ <h3>Building a robot</h3>
       <a name="signup"></a><h2>Want to compete?</h2>
       <div class="column m-8-12 m-offset-2-12 text-left">
         <p>
-          Please register your interest below, we'll let you know when formal entries open later this year.
+          Please register your interest below. We'll let you know once your place is confirmed.
```

Comment:
It might be nice to give an indication of when places will be confirmed, both so that teams know when they'll hear back and so it's fairly clear what the deadline is.

Maybe add something along the lines of "We expect to confirm places at the end of September 2019."?

Reply:
Agreed. I meant to leave a note in the PR description that this needs added.